test/integration/services/patient-quick-search.js

```diff
@@ -226,4 +227,40 @@ context('Patient Quick Search', function() {
     cy
       .go('back');
   });
+
+  specify('No Results with Patient Add', function() {
+    cy
+      .routeSettings(fx => {
+        fx.data.push({ id: 'manual_patient_creation', attributes: { value: true } });
+
+        return fx;
+      })
+      .routesForPatientDashboard()
+      .routeActions()
+      .visit()
+      .wait('@routeActions');
+
+    cy
+      .get('.app-frame__nav')
+      .find('.js-search')
+      .as('search')
+      .click();
+
+    cy
+      .get('.modal')
+      .as('searchModal')
+      .find('.patient-search__input')
+      .type('None')
+      .wait('@routeEmptyPatientSearch');
+
+    cy
+      .get('@searchModal')
+      .should('contain', 'No results match your query')
```

Comment:
Doesn't really matter that much, but could be `No results match your query.` (period included at the end). To match the `i18n` file.